test/features/helpers/files.rb

```diff
@@ -0,0 +1,11 @@
+def get_asset(filename, pdf = false)
```

Comment:
Minor: prefer symbols for kwargs: `def get_asset(filename, pdf: false)` https://thoughtbot.com/blog/ruby-2-keyword-arguments

Reply:
does that change how the function gets called? If so, to what?

Reply:
It would change from `get_asset(image, pdf)` to `get_asset(image, pdf: pdf)`

Reply:
doesn't it make a bit more redundant? or is your point that it becomes more explicit?

Reply:
My initial comment was just that this is a weird way of doing kwargs. I agree you may not want a kwarg here anyway.